src/webgl/shaders/point.vert

```diff
@@ -0,0 +1,11 @@
+attribute vec3 vPosition;
+uniform float vPointSize;
```

Comment:
I would try to keep the naming convention from the other shaders. These should use that first letter to indicate the type of data. So this would be `uPointSize`. [The light vertex shader](https://github.com/processing/p5.js/blob/master/src/webgl/shaders/light.vert) has good naming conventions.

Reply:
Some documentation of the existing shader uniforms [can be found here in webgl_mode_architecture.md](https://github.com/processing/p5.js/blob/ef774d56336eeb9781d340fc069ac2e1e85e26f0/developer_docs/webgl_mode_architecture.md)—and it would be great to add a column for the point shader at the end of this document!